modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics for AdminNetworkPolicy
+
+The following table describes 12 metrics that can support monitoring your ANP and BANP.
```

Comment:
remove this line, redundant with the intro para

Reply:
agree